src/server.ts

```diff
@@ -98,23 +119,12 @@ async function search(
 		res.end(e.message);
 		return;
 	}
-	const criteria: TorrentLocator = pick(data, ["infoHash", "path"]);
-
-	if (
-		!(
-			(criteria.infoHash && criteria.infoHash.length === 40) ||
-			(criteria.path &&
-				criteria.path.length > 0 &&
-				existsSync(criteria.path))
-		)
-	) {
-		logger.info({
-			label: Label.WEBHOOK,
-			message: `Received search request: ${inspect(criteria)}`,
-		});
+	let criteria: TorrentLocator = pick(data, ["infoHash", "path"]);
 
-		const message =
-			"A valid infoHash or accessible path must be provided (infoHash is preferred).";
+	try {
+		criteria = WEBHOOK_SCHEMA.parse(criteria) as TorrentLocator;
+	} catch {
+		const message = `A valid infoHash or accessible path must be provided (infoHash is preferred, https://www.cross-seed.org/docs/basics/daemon#set-up-automatic-searches-for-finished-downloads): ${inspect(criteria)}`;
```

Comment:
```suggestion
		const message = `A valid infoHash or an accessible path must be provided (infoHash is recommended see https://www.cross-seed.org/docs/basics/daemon#set-up-automatic-searches-for-finished-downloads): ${inspect(criteria)}`;
```

Reply:
```suggestion
		const message = `A valid infoHash or an accessible path must be provided (infoHash is recommended: see https://www.cross-seed.org/docs/basics/daemon#set-up-automatic-searches-for-finished-downloads): ${inspect(criteria)}`;
```
probably needs a colon there too.